tests/core/debug/task.yaml

```diff
@@ -0,0 +1,42 @@
+summary: Check that UC debug settings work
+
+details: |
+  Check that UC debug settings enable logging for snapd and
+  setting the log level for systemd as expected.
+
+execute: |
+  if [ "$SPREAD_REBOOT" = 0 ]; then
+      # Remove the variable introduced by prepare.sh
+      sed -i 's/SNAPD_DEBUG=1//' /etc/systemd/system/snapd.service.d/local.conf
+      systemctl daemon-reload
+      systemctl restart snapd
+
```

Comment:
Maybe check that it was effectively removed from the environment, in case we change the way to set it in tests.

Reply:
Done